### PR TITLE
049 hbar transfer demo docs

### LINKED ISSUE #49: Document HBAR transfer Relay interaction steps to a markdown file
### Problem
The relay demo has a few setup steps that are necessary to see a run that highlights the true value of the relay.
Since this can be involving we should document in a separate markdown file for easy reference and reproduction to allow eth devs to hit the ground running

The HBAR transfer flow is the primary scenario

**Scenario 1: Transfer HBAR between 2 users**

- Setup up 2 hedera accounts
- Configure separate wallets to point to a deployed Hedera JSON RPC Relay - could be 2 different relays for same network
- Connect a wallet to each Hedera account
- Send HBAR between users

### Solution
Add documentation highlighting the flow.
New markdown should be placed under `/docs` and README should reference it

Documentation may include

Steps
Code snippets
Screen shots - if applicable

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by mmrt**
**Description**:
Adds documentation of steps for creation of aliased accounts and how to transfer hbar between them.

**Related issue(s)**:

Fixes #49

### CODE REVIEW

File: docs/examples/account-alias-ecdsa.js [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+     */
+
+    console.log('"Creating" a new account');
+
+    const privateKey = PrivateKey.generateECDSA();
+    const publicKey = privateKey.publicKey;
+
+    // Assuming that the target shard and realm are known.
+    // For now they are virtually always 0 and 0.
+    const aliasAccountId = publicKey.toAccountId(0, 0);
```
Comment (Nana-EC) [inline docs/examples/account-alias-ecdsa.js:66]: Add printout of the `0.0.aliasKey` format

Comment (Nana-EC) [reply docs/examples/account-alias-ecdsa.js:66]: If one of the logs below captures this then ignore

File: docs/hbar_transfer.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# Transferring Hbar Using the Hedera JSON-RPC Relay
+
+## Add Hedera Network to Metamask
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:3]: Please add a sentence above noting that any JSON RPC compatible wallet is applicable as they just need to be able to call our endpoints.
Then say for illustration purposes only we are using Metamask as an example

File: docs/hbar_transfer.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Open Metamask and click on the circle in the upper right corner
+2. Select `Settings`->`Networks`->`Add Network`
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:6]: Please add a screenshot of what the populated page looks like with the local node as an example

File: docs/hbar_transfer.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+3. Populate `Network Name` as desired
+4. Fill in `New RPC URL` for the relay you are connecting to
+5. Enter proper `Chain ID`, ensure no error message is seen and that the id matches
+6. Fill in a `Currency Symbol`
+
+## Fund creation account
+In order to get started you will need to have create a Hedera account with sufficient balance to pay for the following transactions.
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:13]:
```suggestion
In order to get started you will need to have created a Hedera account with sufficient balance to pay for the following transactions.
```

File: docs/hbar_transfer.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+You can create an aliased account using the `examples/account-alias-ecdsa.js` located in this directory. You will need to setup your `.env` file with you account id and private key. It is not recommended to run this on mainnet as this will print private keys to the command line. This is for demonstration purposes only.
+
+Record the private keys that are printed to the command line.
+
+## Import Account into Metamask
+
+Repeat this process for every account you wish to import:
+
+1. Copy the private key of an account to be imported
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:26]:
```suggestion
1. Copy the last 64chars of the private key of an account to be imported
```

File: docs/examples/account-alias-ecdsa.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+     * via an AccountInfoQuery.
+     *
+     * Users may continue to refer to the account by its aliasKey AccountId, but they may also
+     * now refer to it by its normal AccountId
+     */
+
+    console.log(`The normal account ID: ${info.accountId.toString()}`);
+    console.log(`The alias key: ${info.aliasKey.toString()}`);
+    console.log(`The alias raw key: ${info.aliasKey.toStringRaw()}`);
+    console.log(`The raw private key: ${privateKey.toStringRaw()}`);
```
Comment (Nana-EC) [inline docs/examples/account-alias-ecdsa.js:105]: Let's add a log that is the print out of the last 64chars of the private key that can be used in wallets.
Could call it the import key

File: docs/hbar_transfer.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+2. Click on the circle in the upper right corner
+3. Select `Import Account`
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:28]: Let's add screen shots of steps 3 & 4 (without the private key of course)

File: docs/hbar_transfer.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+4. Paste your private key into the field and click `Import`
+5. Verify that the balance load successfully
+
+## Send Hbar
+1. Open the receiving account in metamask and copy the address using the button below the account nickname
+2. Go to the sending account and click `Send`
+3. Paste the address into the `Send to` bar
+4. Enter in a valid amount to send to the account into the `Amount` field and click `Next`
+5. Click `Confirm` on the next page
+6. Wait a few seconds for the transaction to confirm and verify the updated balance in the sending/receiving account.
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:38]: Screen shot of account page with pending then successful transaction in "Activity"

File: docs/hbar_transfer.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+1. Copy the private key of an account to be imported
+2. Click on the circle in the upper right corner
+3. Select `Import Account`
+4. Paste your private key into the field and click `Import`
+5. Verify that the balance load successfully
+
+## Send Hbar
+1. Open the receiving account in metamask and copy the address using the button below the account nickname
+2. Go to the sending account and click `Send`
```
Comment (Nana-EC) [inline docs/hbar_transfer.md:34]: Screenshots of different pages. 
Send page, page with address, amount, confirm

File: docs/examples/account-alias-ecdsa.js (hunk 1 above)
Comment (mmrt) [reply docs/examples/account-alias-ecdsa.js:66]: I deleted this as I thought it cluttered up the output which we aren't using anyway. I can add it back in.

File: docs/hbar_transfer.md (hunk 5 above)
Comment (mmrt) [reply docs/hbar_transfer.md:26]: we can't sign with part of a private key. Can you clarify what your concern is?

File: docs/examples/account-alias-ecdsa.js (hunk 6 above)
Comment (mmrt) [reply docs/examples/account-alias-ecdsa.js:105]: changed the message to detail that the raw private key is to be used for importing

File: docs/hbar_transfer.md (hunk 3 above)
Comment (mmrt) [reply docs/hbar_transfer.md:6]: done

File: docs/hbar_transfer.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+## Add Hedera Network to Metamask
+
+1. Open Metamask and click on the circle in the upper right corner
+2. Select `Settings`->`Networks`->`Add Network`
+3. Populate `Network Name` as desired
+4. Fill in `New RPC URL` for the relay you are connecting to
+5. Enter proper `Chain ID`, ensure no error message is seen and that the id matches
+6. Fill in a `Currency Symbol`
+
```
[the remark this answers is not in the document]
Comment (mmrt) [reply docs/hbar_transfer.md]: done

File: docs/hbar_transfer.md (hunk 2 above)
Comment (mmrt) [reply docs/hbar_transfer.md]: done

File: docs/hbar_transfer.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+## Fund creation account
+In order to get started you will need to have create a Hedera account with sufficient balance to pay for the following transactions.
+
+## Create Aliased Hedera Accounts
+Aliased accounts matching the private keys will need to be created in order to properly sign transactions. 
+
+You can create an aliased account using the `examples/account-alias-ecdsa.js` located in this directory. You will need to setup your `.env` file with you account id and private key. It is not recommended to run this on mainnet as this will print private keys to the command line. This is for demonstration purposes only.
```
[the remark this answers is not in the document]
Comment (mmrt) [reply docs/hbar_transfer.md]: fixed

File: docs/hbar_transfer.md (hunk 8 above)
Comment (mmrt) [reply docs/hbar_transfer.md:48]: done

File: docs/hbar_transfer.md (hunk 9 above)
Comment (mmrt) [reply docs/hbar_transfer.md:44]: done

File: docs/examples/.env.sample [hunk 12]
```diff
@@ -0,0 +1,13 @@
+# The operator is responsible for paying transaction and query fees
+#
+# The `LocalWallet` uses the environment variables `OPERATOR_ID`, `OPERATOR_KEY`,
+# and `HEDERA_NETWORK`
+
+# Hedera Operator Account ID
+OPERATOR_ID="0.0.2336"
```
Comment (Nana-EC) [inline docs/examples/.env.sample:7]: Leave all the values empty and note in the doc that they have to fill it in

File: docs/examples/package.json [hunk 13]
```diff
@@ -0,0 +1,12 @@
+{
+    "name": "@hashgraph/examples",
```
Comment (Nana-EC) [inline docs/examples/package.json:2]: nit: 
```suggestion
    "name": "@hashgraph/hedera-json-rpc-relay-examples",
```

File: docs/hbar-transfer.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+## Import Account into Metamask
+
+Repeat this process for every account you wish to import:
+
+1. Copy the raw private key of an account to be imported
+2. Click on the circle in the upper right corner
+3. Select `Import Account`
+
+![Import Account Button](images/importAccountButton.png)
+4. Paste your private key into the field and click `Import`
```
Comment (Nana-EC) [inline docs/hbar-transfer.md:37]: You might need to put a new line after the images since it's making the next text line show up in line

Comment (mmrt) [reply docs/hbar-transfer.md:38]: Interesting the intelliJ viewer isn't rendering this way. Fixed

**Pull request merged**